README clarification (#61)
lagonapp/serverless: README.md

## About

> Lagon is a free Open Source Runtime and SaaS that make it easy to deploy TypeScript and JavaScript Serverless Functions at the Edge, using V8 Isolates.  The goal of Lagon is to be an alternative to CloudFlare Workers® and Deno Deploy that you can self-host.

Reviewer: To be more precise, Lagon will also be available as a Cloud version so you don't have to manage servers yourself:

```suggestion
Lagon is a free Open Source Runtime and SaaS that makes it easy to deploy TypeScript and JavaScript Serverless Functions at the Edge, using V8 Isolates. The goal of Lagon is to be an alternative to Cloudflare Workers and Deno Deploy, with both Cloud and self-hosted versions.
```
